complex/ft_comp.c

```diff
@@ -54,8 +54,6 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
-	opts.comp_method = test_info.cq_wait_obj;
-
```

Comment:
Please take a look at ft_cq_set_wait_attr() and ft_cntr_set_wait_attr() to see how comp_method is used to select the cq/cntr wait object. The FT_... enums are used to configure the cq/cntr attributes.

I'm still not following what the *intent* was behind this change.  That's what I'm trying to figure out.  :)